Update inferred-service-opt-in.md (#22024)
DataDog/documentation: content/en/tracing/guide/inferred-service-opt-in.md

{{< /code-block >}}

### OpenTelemetry Collector 

Minimum version recommended: opentelemetry-collector-contrib >= [v0.95.0][7].

Example [collector.yaml][6].

{{< code-block lang="yaml"  collapsible="true" >}}

connectors:
  datadog/connector:
    traces:
      compute_stats_by_span_kind: true
      peer_tags_aggregation: true
      peer_tags: ["_dd.base_service","amqp.destination","amqp.exchange","amqp.queue","aws.queue.name","bucketname","cassandra.cluster","db.cassandra.contact.points","db.couchbase.seed.nodes","db.hostname","db.instance","db.name","db.system","grpc.host","hazelcast.instance","hostname","http.host","messaging.destination","messaging.destination.name","messaging.kafka.bootstrap.servers","messaging.rabbitmq.exchange","messaging.system","mongodb.db","msmq.queue.path","net.peer.name","network.destination.name","peer.hostname","peer.service","queuename","rpc.service","rpc.system","server.address","streamname","tablename","topicname"]

{{< /code-block >}}

> If your collector version is below [v0.95.0][7], you need to configure the exporter with the peer tags config:

Reviewer: ```suggestion
If your collector version is below [v0.95.0][7], use an exporter configuration with the following `peer_tags`:
```

> ### Datadog Exporter configuration

Reviewer: ```suggestion
```
It's a little confusing to go to a new section here from the previous sentence, so I've removed the header and incorporated it into the previous sentence.

[3]: /tracing/service_catalog/
[4]: https://github.com/DataDog/datadog-agent/releases/tag/7.50.3
[5]: /agent/guide/agent-configuration-files/?tab=agentv6v7

> [6]: [https://github.com/open-telemetry/opentelemetry-collector-contrib/blob/main/exporter/datadogexporter/examples/collector.yaml#L328-L341](https://github.com/open-telemetry/opentelemetry-collector-contrib/blob/main/exporter/datadogexporter/examples/collector.yaml#L335-L357)https://github.com/open-telemetry/opentelemetry-collector-contrib/blob/main/exporter/datadogexporter/examples/collector.yaml#L335-L357

Reviewer: ```suggestion
[6]: https://github.com/open-telemetry/opentelemetry-collector-contrib/blob/main/exporter/datadogexporter/examples/collector.yaml#L335-L357
```
